text/1071-cell.md

```diff
@@ -84,7 +84,7 @@ interface Cell<Value> extends Reactive<Value> {
     /**
     * Utility to create a Cell without the caller using the `new` keyword.
     */
-    static create<T>(initialValue: T, equals?: (a: T, b: T) => boolean): Cell<T>;
+    static create<T>(initialValue: T, equals?: (a: T, b: T) => boolean, description?: string): Cell<T>;
```

Comment:
in 99% cases quality operator will be default, should description will be second argument?
Another option, create Cell dto, to set default values.
Like: 
```ts
const Cel = CellDto.fromEquals((a,b)=> a !== b);
Cell.create('name', 'debug name');


```

Reply:
I'll have to test the perf of this, because right now I can't really make a decision on this as I don't know how it'll affect things.